projects/storefrontlib/shared/components/bind-attributes/bind-attributes.directive.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AfterViewInit,
+  Directive,
+  ElementRef,
+  Input,
+  Renderer2,
+} from '@angular/core';
+
+@Directive({
+  selector: '[cxBindAttributes]',
+})
+export class BindAttributesDirective implements AfterViewInit {
+  /**
+   * Usage:
+   * [cxBindAttributes]="{
+      'aria-label': selectedLabel,
+      'aria-controls': ariaControls
+     }"
+   */
+  @Input() cxBindAttributes: { [attribute: string]: any };
+
+  /* DOMString that matches the element to bind the attributes */
+  @Input() target: string;
```

Comment:
what about `cxBindTarget`?

Reply:
I like the first one more since `cxBindAttributesTarget` is a bit long

Reply:
> I like the first one more since `cxBindAttributesTarget` is a bit long
long is not a defect if it's more precise, and here it could be a prefix of the directive which point to which one's part it is